segmentstore/server/src/test/java/io/pravega/service/server/containers/StreamSegmentContainerTests.java

```diff
@@ -104,10 +104,10 @@
  * DurableDataLog.
  */
 public class StreamSegmentContainerTests extends ThreadPooledTestSuite {
-    private static final int SEGMENT_COUNT = 1;
+    private static final int SEGMENT_COUNT = 100;
```

Comment:
This and line 110 were changed inadvertently a couple of months back. This is just reverting them back to their original values.